great_expectations/rule_based_profiler/data_assistant/volume_data_assistant.py

```diff
@@ -233,7 +233,7 @@ def _build_categorical_columns_rule() -> Rule:
             "strict_min": False,
             "strict_max": False,
             "false_positive_rate": 0.05,
-            "estimator": "bootstrap",
+            "estimator": "exact",
```

Comment:
@NathanFarmer I would prefer that we leave `bootstrap` as is.  `DataAssistantRunner.run()` already sets `exact` by default.

Reply:
@alexsherstinsky we should use `quantiles` here if the behavior has changed in order to achieve parity between `OnboardingDataAssistant` and `VolumeDataAssistant`.

Reply:
✅ after our offline conversation